src/cljs/athens/util.cljs

```diff
@@ -92,3 +95,71 @@
       (string/replace x #"PM" "pm"))))
 
 
+;; -- Linked & Unlinked References ----------
```

Comment:
I think `db` is a slightly better place to put this than `util`, but neither are great.

Reply:
I agree! We can just put it here for now, and potentially create a new references.cljs when we get started on graph previews.